.github/workflows/autocomment-on-events.yml

```diff
@@ -0,0 +1,113 @@
+name: Auto Comment on Events
+
+on:
+  issues:
+    types: [opened, closed]
+  pull_request_target:
+    types: [opened, closed]
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  # Job for commenting on opened issues
+  comment-on-issue-open:
+    if: github.event_name == 'issues' && github.event.action == 'opened'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add Comment to Opened Issue
+        run: |
+          COMMENT=$(cat <<EOF
+          {
+            "body": "Thank you for creating this issue! 🎉 We'll look into it as soon as possible. In the meantime, please make sure to provide all the necessary details and context. If you have any questions or additional information, feel free to add them here. Your contributions are highly appreciated! 😊\n\nYou can also check our [CONTRIBUTING.md](https://github.com/dmotts/yt-to-gdrive-uploader/blob/main/CONTRIBUTING.md) for guidelines on contributing to this project."
```

Comment:
replace `https://github.com/dmotts/yt-to-gdrive-uploader/blob/main/CONTRIBUTING.md` with `https://github.com/ai-buddies/ai-buddies.github.io/blob/main/CONTRIBUTING.md`